packages/modelviewer.dev/rollup.config.js

```diff
@@ -72,6 +72,78 @@ const outputOptions = [
     plugins,
     onwarn,
   },
+  {
+    input: './lib/docs-and-examples/documentation-main.js',
```

Comment:
Instead of doing all of these separately, can we roll them into a single bundle? That's what rollup is for after all. Take a look at any of the other packages to see how it's done.

Reply:
yep, done